### DISCUSSION TITLE
Remove strange black line on top of dashboard.nvim

### DISCUSSION THREAD

**Discussion by Anifyuli** [LazyVim/LazyVim#2341] [Q&A] [answered]
Hello anyone. I confused for removing strange horizontal black line on top of dashboard.nvim, because it messing around. 
You can see in this screenshot
![image](https://github.com/LazyVim/LazyVim/assets/73734843/4aed81e5-e185-483d-8338-15fec3f03555)

I add this to my config but it not removed anymore.
```lua
hide = {
        -- this is taken care of by lualine
        -- enabling this messes up the actual laststatus setting after loading a file
        statusline = true,
        tabline = true,
        winbar = true,
      },
```

How must I do to remove it? Thanks for your help and sorry My English

**Comment by dpetka2001** [comment]
I believe this comes from bufferline. By default LazyVim has this feature set to false as you can see in the [docs](https://www.lazyvim.org/plugins/ui#bufferlinenvim), the feature is `always_show_bufferline = false`. In your personal configuration you must have changed this to `true` in some place.

**Reply by Anifyuli** [reply]
Yes, I add `always_show_bufferline` to my configs. So, I must disable it to remove this strange horizontal black line?

**Reply by dpetka2001** [reply]
Yes that black line is caused by that setting. Try disabling it to see if it solves your problem.

**Reply by Anifyuli** [reply]
I see. So I can't always show my bufferline and hide that strange line?

**Reply by dpetka2001** [reply]
That strange line is your bufferline. What do you mean? It just doesn't show any buffers on the bufferline when you're on the dashboard.

**Reply by Anifyuli** [reply]
I mean I can't always show my bufferline if I edit a one file if I hide it. Like this.
![image](https://github.com/LazyVim/LazyVim/assets/73734843/d7d7b7b1-7882-4ee3-af32-cf503dc83b33)
But I want to show bufferline if I edit only a single file like that, and I hide that strange line.
![image](https://github.com/LazyVim/LazyVim/assets/73734843/04fcfc77-872d-43a0-9085-0becff2d8857)

**Reply by dpetka2001** [reply]
That strange line is your bufferline. If you want to always show it, then it will be there as you see it. Nothing can be done about it.

**Reply by Anifyuli** [reply]
Oh, I see. No any choice without that two choices. Thanks for your help

**Comment by Anifyuli** [comment, marked as answer]
Solved : Just change `bufferline.nvim` event from `VeryLazy` to `BufEnter` and define `always_show_bufferline = true` in it config file
